src/components/Tag/Tag.tsx

```diff
@@ -10,16 +10,17 @@ export interface TagProps extends Omit<React.HTMLAttributes<HTMLSpanElement>, 's
   type?: TagType
   style?: ExternalStyles
   icon?: Icons
+  height?: String
   removable?: boolean
   onRemove?(): void
 }
 
 export function Tag(props: TagProps) {
-  const { type, style, children, icon, removable, onRemove, ...rest } = props
+  const { type, style, children, icon, height, removable, onRemove, ...rest } = props
   const { classes, css } = useStyles(createStyles)
 
   return (
-    <span className={css(classes.tag, classes[type], style)} {...rest}>
+    <span className={css(classes.tag, classes[type], style, `height: ${height}`)} {...rest}>
```

Comment:
I think that the common thing to do is to set this attribute on tag css directly, you can pass to the `createStyles` function by just adding a new parameter (height) on `useStyles` at line 20.